charts/incubator/signal-cli-rest-api/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+appVersion: 1.0.0
+description: Dockerized Signal Messenger REST API
+name: signal-cli-rest-api
+version: "0.57"
```

Comment:
This should be the chart version not the app version

Reply:
appversion and version should be fixed now. Thanks!